components/webui/imports/api/ingestion/server/CompressionDbManager.js

```diff
@@ -59,7 +59,9 @@ class CompressionDbManager {
                 (
                     SELECT *
                     FROM SelectedColumns
-                    WHERE _id=${jobId}
+                    WHERE 
+                        _id=${jobId} && 
+                        ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.UPDATE_TIME} >= '${lastUpdateDate}'
```

Comment:
Instead of comparing against a string date, we can pass in a timestamp and use `FROM_UNIXTIME()`? 
ref: [FROM_UNIXTIME(unix_timestamp[,format])](https://dev.mysql.com/doc/refman/8.4/en/date-and-time-functions.html#function_from-unixtime)